languages/js/src/Query.ts

```diff
@@ -38,6 +38,10 @@ export class Query {
     this.results = this.start();
   }
 
+  public source(): string {
```

Comment:
Also please add API docs -- see the rest of the methods in this file for examples. This one should probably be marked `@internal`

Reply:
Now that I look at it more closely, we aren't actually using this method at all. You're calling `.source()` directly on the FFI query instead of on the TS `Query` instance.